db/migrate/20230719092447_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+
+      t.timestamps
+  end
+end
```

Comment:
- [ ] Also here, kindly make sure to add all the columns as suggested by the last reviewer.  Right now, you are missing columns for `comments_counter`, `likes_counter` and a reference to the `users table`. Kindly make sure to implement this also ✨📌

```
  create_table :posts do |t|
    t.string :title
    t.text :text
    t.integer :comments_counter
    t.integer :likes_counter
    t.timestamps 
    t.references :author, foreign_key: { to_table: 'users' }
  end
```